paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -209,6 +211,20 @@ internal fun AddPaymentMethodWithPaymentElement(
 ) {
     val horizontalPadding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
+    SimpleDialogElementUI(
+        openDialog = viewState.displayConfirmationModal,
+        titleText = stringResource(id = R.string.stripe_confirm_close_form_title),
+        messageText = stringResource(id = R.string.stripe_confirm_close_form_body),
+        confirmText = stringResource(id = R.string.stripe_paymentsheet_close),
```

Comment:
Nit: I feel like we should use something like `Discard changes` or similar, which is more specific to the consequences of the action.

We should also use a different button color to highlight the destructive action.

Reply:
This is what we have in PaymentSheet as well. I am also following what iOS has.